### PR TITLE
Feature/offpay 487

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

**Comment by peteanning** [comment]
looking at this for you

File: app/uk/gov/hmrc/offpayroll/models/ExitFlow.scala [hunk 1]
```diff
@@ -28,7 +28,7 @@ object ExitFlow extends Webflow {
 
   override def getNext(currentElement: Element): Option[Element] = getNext(currentElement, clusters(0))
 
-  override def getStart(): Element = getElementById(0, 0).get
+  override def getStart(interview: Map[String, String]): Element = getElementById(0, 0).get //fixme refactor to use getStart ????
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/models/ExitFlow.scala]: Habeeb think you meant to fix this?

File: app/uk/gov/hmrc/offpayroll/models/SetupFlow.scala [hunk 2]
```diff
@@ -29,7 +29,7 @@ object SetupFlow extends Webflow {
 
   override def getNext(currentElement: Element): Option[Element] = getNext(currentElement, setupCluster)
 
-  override def getStart(): Element = setupCluster.clusterElements(0)
+  override def getStart(interview: Map[String, String]): Element = setupCluster.clusterElements(0)
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/models/SetupFlow.scala]: Should be getStart(interview) ?

File: test/uk/gov/hmrc/offpayroll/models/PersonalServiceClusterSpec.scala [hunk 3]
```diff
@@ -63,11 +63,123 @@ class PersonalServiceClusterSpec extends FlatSpec with Matchers with ClusterSpec
...
+      "personalService.workerSentActualSubstitute" -> "personalService.workerSentActualSubstitute.noSubstitutionHappened",
+      "personalService.possibleSubstituteRejection" -> "Yes",
+      currentQnA)
+
+    val maybeElement = personalServiceCluster.shouldAskForDecision(previousAnswers, currentQnA)
+
+    maybeElement.isEmpty shouldBe true
+  }
+
+  it should " ask possibleSubstituteRejection when 'Yes' is the answer to setup.hasContractStarted" in {
```
Comment (peteanning) [inline test/uk/gov/hmrc/offpayroll/models/PersonalServiceClusterSpec.scala]: This test seems to contradict the name "ask possibleSubstituteRejection when 'Yes' is the answer to setup.hasContractStarted"
but this variable partialInterview_hasContractStarted_No ??

File: app/uk/gov/hmrc/offpayroll/models/SetupFlow.scala (hunk 2 above)
Comment (habeeb01) [inline app/uk/gov/hmrc/offpayroll/models/SetupFlow.scala]: setupCluster.clusterElements(0) is the start for this flow. Don't think this needs to change

File: app/uk/gov/hmrc/offpayroll/models/ExitFlow.scala (hunk 1 above)
Comment (habeeb01) [inline app/uk/gov/hmrc/offpayroll/models/ExitFlow.scala]: I have removed the comment and explicitly named the exitCluster as the starting point in this flow

File: test/uk/gov/hmrc/offpayroll/models/PersonalServiceClusterSpec.scala (hunk 3 above)
Comment (habeeb01) [inline test/uk/gov/hmrc/offpayroll/models/PersonalServiceClusterSpec.scala]: I have corrected the name of the test. The test itself was already correct

File: app/uk/gov/hmrc/offpayroll/models/SetupFlow.scala (hunk 2 above)
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/models/SetupFlow.scala]: Shouldn't you delegate to the first Cluster in the flow to be consistent as you do in the OffPayrollWebFlow L58 clusters.head.getStart(interview)

so setupCluster.getStart(interview)

**Pull request merged**